server/src/models/index.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const uri = 'mongodb://localhost/group_tripper';
```

Comment:
This should be pulled out into the .env file

Reply:
is this ok ? 
const uri = process.env.mongo_uri || 'mongodb://localhost/group_tripper';